book/client.md

```diff
@@ -19,11 +19,30 @@ The first thing to do is set up error handling using `error-chain`. I have
 $ cargo add error-chain
 ```
 
-You'll then need to add the corresponding `extern crate` statement to `lib.rs` 
-and create an `errors.rs` module. While you're at it, you may also want to add 
-the `reqwest` and `cookie` crates.
+You'll then need to add the corresponding `extern crate` statement to `lib.rs`. 
+While you're at it, add also the `reqwest`, `cookie`, `chrono`, `fern`, `log` 
```

Comment:
This is a good idea. I don't think I mention later on when I've added extra dependencies, meaning it'll probably be a bit confusing for someone following along at home.

Reply:
Yes, I was quite lost a few times when with the next content added to my files and mentally digested on building the log was filled with errors :)
Guess I made and do make a good sample for a Rust beginner who needs hand-holding/documentation for every little step :) (so far had only walked through the Rust Programming book 1rst edition many months ago, and then forgotten most of it).
Of course trying to understand the issues etc. also yielded some knowledge back, but I doubt this was your intention :)